presto-tests/src/test/java/com/facebook/presto/tests/TestAfterBurnerSupport.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.tests;
+
+import com.facebook.presto.tests.tpch.TpchQueryRunnerBuilder;
+import org.testng.annotations.Test;
+
+public class TestAfterBurnerSupport
+{
+    @Test
+    public void smokeTestAfterBurnerSupport()
+            throws Exception
+    {
+        try (DistributedQueryRunner queryRunner = TpchQueryRunnerBuilder.builder()
+                .setSingleExtraProperty("experimental.json-serde-codegen-enabled", "true")
```

Comment:
This test doesn't really test the afterburner module as it creates `TestingPrestoServer` instances under the hood, and `TestingPrestoServer` doesn't install the afterburner module.